### PR TITLE
Yatch house front end  project

### PR DESCRIPTION
**Pull request by Baroka-wp**
In this PR we have created a full stack house reservation project. The project is divided in a backend part made with Ruby on rails and Postgresql and a frontend part made with Rect and Redux.
Here is the link to the front backend directory:
[back-end repos](https://github.com/Baroka-wp/yatch_house/pull/36)

### Live Demo

[go live](https://yatch-house.netlify.app/)

### Get started 
- Open terminal and clone the project: 
```
git clone https://github.com/baroka-wp/yatch-house-front-end.git
```
```
cd yatch-house-front-end
```
- Install package

```
 npm install
```
- To run the app in the development mode : 
```
npm start
```

- Launch the test runner :
```
 npm test
```
In case you run into any errors please raise an [issue](https://github.com/baroka-wp/yatch-house-front-end/issues).

### CODE REVIEW

File: src/components/MainNavBar.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    }
+    return false;
+  };
+
+  const handleLogout = async () => {
+    // try {
+    //   await axios.delete('http://localhost:3001/users/sign_out');
+    // } catch (err) {
+    //   console.log(err);
+    // }
```
Comment (Meltrust) [inline src/components/MainNavBar.js:24]: - [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍

File: src/components/MobileNavBar.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+
+  const openModal = () => {
+    setIsOpen(true);
+  };
+
+  // const afterOpenModal = () => {
+  //   // references are now sync'd and can be accessed.
+  //   subtitle.style.color = '#f00';
+  // };
```
Comment (Meltrust) [inline src/components/MobileNavBar.js:32]: - [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍

File: src/components/pages/AddHouse.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// import { Navigate } from 'react-router';
```
Comment (Meltrust) [inline src/components/pages/AddHouse.js:2]: - [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍

**Review by Meltrust — COMMENTED** [review]
Hi @Baroka-wp,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  App is simple, but works ✔️ 
-  Good readme ✔️ 

You can do it

## Suggested Changes ♻️
_Check the comments under the review._

You can use as many of my suggestions as you want.  If there is anything you would like to skip - feel free to do that.  However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/components/pages/NewReservation.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                  <div className="end-date">
+                    <p>End Date</p>
+                    <DatePicker
+                      selected={endDate}
+                      onChange={(date) => {
+                        setEndDate(date);
+                        calculateTotalPriceFromEnd(date);
+                      }}
+                      minDate={new Date()}
+                    />
```
[the remark this answers is not in the document]
Comment (GedeonTS) [reply src/components/pages/NewReservation.js:144]: Thank you @Meltrust, on chrome it's displaying well but we fixed it for all others too.

**Comment by mcrd25** [comment]
Hi Team

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**